scout/quickstart.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Docker Scout quickstart
+keywords: scout, supply chain, vulnerabilities, packages, cves, scan, analysis, analyze
+description: >
+  Learn how to get started with Docker Scout to analyze images and fix vulnerabilities
+redirect_from:
+  - /atomist/get-started/
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout analyzes image contents and generates a detailed report of packages
+and vulnerabilities that it detects. It can provide you with
+suggestions for how to remediate issues discovered by image analysis.
+
+This guide takes a vulnerable container image and shows you how to use Docker
+Scout to identify and fix the vulnerabilities, compare image versions over time,
+and share the results with your team.
+
+## Setup
+
+[This example project](https://github.com/docker/scout-demo-service) contains 
+a vulnerable Node.js application that you can use to follow along.
+
+1. Clone its repository:
+
+    ```shell
+    git clone https://github.com/docker/scout-demo-service.git
+    ```
```

Comment:
Change to `console`, add prompt, and indent by three spaces. Here and all other occurrences

````suggestion
   ```console
   $ git clone https://github.com/docker/scout-demo-service.git
   ```
````